packages/lab/src/Canvas/stories/Canvas.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import { useEffect, useState } from "react";
+import { css } from "@emotion/css";
+import { Meta, StoryObj } from "@storybook/react";
+import { BackgroundVariant } from "reactflow";
+import {
+  HvButton,
+  HvDropDownMenu,
+  HvIconButton,
+  HvInlineEditor,
+  HvLoading,
+  theme,
+} from "@hitachivantara/uikit-react-core";
+import {
+  Backwards,
+  Calendar,
+  DataSource,
+  Plane,
+  Pod,
+  Redo,
+  Schema,
+  Undo,
+  User,
+} from "@hitachivantara/uikit-react-icons";
+import {
+  HvCanvasPanel,
+  HvCanvasTab,
+  HvCanvasToolbar,
+  HvFlow,
+  HvFlowBackground,
+  HvFlowMinimap,
+} from "@hitachivantara/uikit-react-lab";
+
+import { ListView } from "./ListView/ListView";
+import { eds, nds, nodeTypes } from "./nodes";
+import { TreeView } from "./TreeView";
+
+const meta: Meta = {
+  title: "Lab/Canvas",
+};
+export default meta;
+
+export const Main: StoryObj = {
```

Comment:
I think this sample could be added to Chromatic and we should leverage `play` to open the panel